src/js/react/apps/maternity-and-child-health-clinic-search/components/ResultCard.tsx

```diff
@@ -13,10 +22,39 @@ const ResultCard = ({ address, name, name_override, picture_url, provided_langua
   let cardImage;
 
   if (imageOverride) {
-   cardImage = <CardImage
+    const sources = [
```

Comment:
Combine the sources variables in to a one template

Reply:
I combined all of them :) So this is fixed 🦖